setup.cfg

```diff
@@ -35,22 +36,16 @@ install_requires =
     prometheus_client
     protobuf
 
-tests_require =
-    # Testing dependencies
+[options.extras_require]
+Testing =
     pytest
     pytest-cpp
     pytest-timeout
-    flask
     future
     pytest-server-fixtures
-    six
     mock
     boto3
-    arctic
-    kafka-python
-    kazoo
-    hypothesis
     moto
-
-[tool:pytest]
-cpp_files=cpp/tests/*
+    flask  # Used by moto
```

Comment:
(nit) I think `moto[server]` should pick up the flask stuff and leave it a bit more obvious why we need it at all.